src/test/HST/Util/MessagesTests.hs

```diff
@@ -0,0 +1,126 @@
+module HST.Util.MessagesTests ( testMessages ) where
```

Comment:
It's strange that this module is marked as completely new in GitHub. It exists in the `main`-branch in an updated version.